lib/core/Site.js

```diff
@@ -159,6 +159,9 @@ class Site extends React.Component {
                 }}
               />
             ))}
+          {process.env.NODE_ENV !== 'production' && (
```

Comment:
Instead of checking whether it's `production`, check whether it's `development`.